File: utils/logger.py
import logging
import sys

# Configuration optimisée du logging
logging.basicConfig(
    level=logging.INFO,  # Changé de DEBUG à INFO pour de meilleures performances
    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
    handlers=[
        logging.StreamHandler(sys.stdout)
    ]
)

logger = logging.getLogger(__name__)
logger.setLevel(logging.INFO)  # Niveau explicite

File: utils/logger.js
/**
 * Simple logger utility
 */
class Logger {
    info(message) {
        console.log(`[INFO] ${new Date().toISOString()} - ${message}`);
    }

    error(message) {
        console.error(`[ERROR] ${new Date().toISOString()} - ${message}`);
    }

    warn(message) {
        console.warn(`[WARN] ${new Date().toISOString()} - ${message}`);
    }

    debug(message) {
        console.log(`[DEBUG] ${new Date().toISOString()} - ${message}`);
    }
}

export const logger = new Logger();